### ISSUE TITLE
Input not working while Netplay host is active

### ISSUE THREAD

**Issue by vigorousjammer** [libretro/RetroArch#6517]
## Description

When I start a Netplay host, my inputs don't work until I stop the Netplay host.

### Expected behavior

Inputs should be working whether Netplay host is active or not.

### Actual behavior

When I choose "start netplay host", and then launch a game, it detects the XInpit controller in Port #0 
However, none of the buttons are working when I try to control the game.
I can open the in-game overlay and control that fine, but can't control the game whatsoever until I stop the Netplay host. However, as soon as I turn it back on they stop working again.

It should also be mentioned I am not using a relay server.

### Steps to reproduce the bug

1. Launch RetroArch
2. Choose "Start Netplay Host"
3. Launch any game
4. Attempt to control the game, and fail to do so
5. Open the in-game overlay and Stop netplay host
6. Inputs are now working as expected while not hosting netplay
7. Open the overlay and re-start the netplay host
8. Inputs are no longer working

### Bisect Results

After updating to v1.7.1

### Version/Commit
You can find this information under Information/System Information

- RetroArch: 1.7.1

### Environment information

- OS: Windows 10 64-bit
- Compiler: MinGW (6.3.0) 64-bit

**Comment by Tatsuya79** [comment]
I didn't have that issue starting netplay using snes9x with Super Mario World on win 7 x64, xinput, recent nightly.
Which game and core are you experiencing the problem with?
If you compiled RetroArch yourself, try the buildbot build.

**Comment by fr500** [comment]
If the controller is set to UNKNOWN before starting netplay this happens.

Make sure you have your controller set to SNES GamePad

**Comment by vigorousjammer** [comment]
@Tatsuya79 I was testing it with snes9x 2010 core v1.52.4 32a479c (not sure if that's the latest one, but it's the only one listed in the cores download menu)
The roms I used were Pocky & Rocky as well as Gradius III. 
I also tested it with the FCEUmm core with Adventure Island II and had the same issue.
My build isn't the nightly, however, but the last stable release, 1.7.1
As far as I know I'm just using the default launcher, and I definitely didn't compile it myself, just used the standard Windows installer.

@fr500 Before starting Netplay, the controller is set as "RetroPad", but once I launch the game & the core, it switches to "SNES Joypad".
I also just tested it by starting the core & the rom before launching NetPlay, double checking that it was set to "SNES JoyPad" before doing so, and still had the same issue.

**Comment by vigorousjammer** [comment]
Well, I must have installed a weird version or something. I re-downloaded RetroArch, this time downloading the 7zip extractable version, I extracted it into a new folder and everything now works flawlessly.
Not sure what I did earlier. I may have used the .exe to install new files on top of an old version without realizing, as right before this, I had an issue with input because I installed the version from the "Windows 2010" folder on Libretro instead of just the "Windows" folder.
I figured the new installer would just overwrite any files from the other version, but I guess in retrospect I should have deleted the incorrect version before installing the other one.

In any case, everything is working now. Thanks both of you for your help, even though it was just a stupid mistake on my part.
